packages/themes/teas/CHANGELOG.md

```diff
@@ -0,0 +1 @@
+# @sl-design-system/teas
```

Comment:
Not sure if you need to create this manually. Usually the changes bot does it for us.

Reply:
Yes